app/routes/app+/rewards.addresses.tsx

```diff
@@ -243,7 +237,11 @@ function RemoveAddressButton({ wallet }: { wallet: WalletWithChain }) {
             <p className="text-sm font-semibold border-solid border-0 border-r border-trueGray-200 p-3">
               {wallet.networkName}
             </p>
-            <p className="pl-2 overflow-clip">{wallet.address}</p>
+            <p className="pl-2">
+              {wallet?.address && wallet?.address.length < 30
```

Comment:
I think we have a helper for this. Side note: would be nice to have a component with the "click to copy" functionality for all the wallet addresses we display on the site.